src/EncompassRest/Loans/Enums/LienType.cs

```diff
@@ -0,0 +1,10 @@
+using System.Runtime.Serialization;
+
+namespace EncompassRest.Loans.Enums
+{
+    public enum LienType
+    {
```

Comment:
we need to be careful with this one.  if people pull it from the database, it may be "First Lien" and "Second Lien"

Reply:
How would users pull a `Loan` object from the database?

Reply:
using the pipeline cursor.  this may be something that im confusing, but i remember there being issues with this field

Reply:
But you can't generate a `Loan` object which is where this enum will be used from a pipeline cursor.

Reply:
ok.  never mind. i cannot come up with a scenario where this would be an issue.